src/kriscv/kdist/riscv-semantics/sparse-bytes.md

```diff
@@ -76,6 +76,13 @@ We provide helpers to prepend either data or an empty region to an existing `Spa
   rule readByteBF(#bytes(B) EF, I) => readByteEF(EF, I -Int lengthBytes(B))
                                              requires I >=Int lengthBytes(B)
 ```
+For symbolic execution, we need to tackle the patterns of `#bytes(B +Bytes _) _` and `#bytes(B +Bytes BS) EF` to obtain as exact as possible values for `readByte`.
```

Comment:
These lemmas should be moved into their own K file.

See e.g. how `evm-semantics` is organized: https://github.com/runtimeverification/evm-semantics/tree/master/kevm-pyk/src/kevm_pyk/kproj/evm-semantics/lemmas